src/test/ci/command_prechecks.test.ts

```diff
@@ -494,187 +495,19 @@ describe("command prechecks", () => {
-                it("should return true", () => {
-                    assert.strictEqual(
-                        CommandPrechecks.answerHiddenPrecheck({
-                            session,
-                            messageContext,
-                            parsedMessage: {
-                                components: [],
-                                action: "answer",
-                                argument: "",
-                                message: "",
-                            },
-                        }),
-                        true,
-                    );
-                });
-            });
-
-            describe("change to multiple choice via explicit multiple choice", () => {
-                it("should return true", () => {
-                    assert.strictEqual(
-                        CommandPrechecks.answerHiddenPrecheck({
-                            session,
-                            messageContext,
-                            parsedMessage: {
-                                components: ["easy"],
-                                action: "answer",
-                                argument: "",
-                                message: "",
-                            },
-                        }),
-                        true,
-                    );
-                });
-            });
-
-            describe("change to typing typos", () => {
-                it("should return true", () => {
-                    assert.strictEqual(
-                        CommandPrechecks.answerHiddenPrecheck({
-                            session,
-                            messageContext,
-                            parsedMessage: {
-                                components: ["typingtypos"],
-                                action: "answer",
-                                argument: "",
-                                message: "",
-                            },
-                        }),
-                        true,
-                    );
-                });
-            });
-
-            describe("changing non-answer option", () => {
-                it("should return true", () => {
-                    assert.strictEqual(
-                        CommandPrechecks.answerHiddenPrecheck({
-                            session,
-                            messageContext,
-                            parsedMessage: {
-                                components: [],
-                                action: "timer",
-                                argument: "",
-                                message: "",
-                            },
-                        }),
-                        true,
-                    );
-                });
-            });
-        });
-    });
-
     describe("timerHiddenPrecheck", () => {
         const GUILD_ID = "123";
 
-        describe("hidden game session", () => {
+        describe("hidden answer type", () => {
```

Comment:
This test is no longer relevant

Reply:
I think it still is, we prevent disabling the timer when hidden is enabled since it requires it